app/src/components/modal/common_styled/index.tsx

```diff
@@ -68,16 +64,8 @@ export const BalanceItemSide = styled.div`
   align-items: center;
 `
 
-export const BalanceItemTitle = styled.p<{ notSelected?: boolean }>`
-  font-size: ${props => props.theme.fonts.defaultSize};
+export const BalanceItemTitle = styled(TYPE.bodyRegular)<{ notSelected: boolean }>`
   color: ${props => (props.notSelected ? props.theme.text2 : props.theme.text1)};
-  margin: 0;
-`
-
-export const BalanceItemBalance = styled.p`
-  font-size: ${props => props.theme.fonts.defaultSize};
-  color: ${props => props.theme.text2};
-  margin: 0;
```

Comment:
Also since we now only have one use for balanceItemTitle and its seems redundatnt we should consolidate these 2 classes into one. And name it something more general like that would make more sense with new addtions.  Like maybe DataItemLight or something like that, im shit at naming....

Reply:
I looked into it and it's quite impossible because we would loose the hover effect, or at least, the hover effect would also apply to the balances of the item. Let's discuss that on a quick call with the files in front of us...